files/opt/dash-button/scripts/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "buttons": [
+    {
+      "mac": "xxxxx",
+      "provider": "bring",
+      "providerConfig": {
+        "listId": "xxxxx",
+        "productName": "Ländle Milch"
+      }
+    },
+    {
+      "mac": "xxxxx",
+      "provider": "bring",
+      "providerConfig": {
+        "listId": "xxxxx",
+        "productName": "Laktosefreie Milch"
+      }
+    }
+  ],
+  "providerConfig": {
+    "bring": {
+      "notificationUserId": "xxxxx",
+      "notificationApiKey": "xxxxx",
+      "notificationUserUuid": "xxxxx",
+      "notificationSender": "xxxxx"
+    }
+  }
+}
```

Comment:
Die Konfigurationsdatei sollte meiner Meinung nach nicht in `files` liegen, da sie nicht mit in das Docker-Image _gebuildet_ werden sollte. Sie soll ausschließlich gemounted werden.

Vorschlag: Konfigurationsdatei liegt als `config.example.json` im Hauptordner des Repositories.

Darüber hinaus bitte noch darauf achten, dass alle Dateien **mit einem Zeilenumbruch enden**.